_includes/samples/PriceFeeds/PriceConsumerWithSequencerCheck.sol

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Is it possible to test this code on a testnet?

Reply:
Not at this time.